CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.6.0 (2019-05-03)
+
+- Added mixins for responsive designs. [#58](https://github.com/blackbaud/skyux-theme/pull/58)
+- Fixed radio and checkbox labels to have correct spacing. [#62](https://github.com/blackbaud/skyux-theme/pull/62)
```

Comment:
Sorry, I don't know why I didn't include this in the first comment. 😬 

Fixed spacing for radio and checkbox labels.

Reply:
No problem. Done.